src/module/actor/party/document.ts

```diff
@@ -22,6 +24,11 @@ class PartyPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e | n
         return [...this.baseAllowedItemTypes, ...(this.campaign?.extraItemTypes ?? [])];
     }
 
+    /** Parties use the campaign's level as their own level, as they otherwise don't have a level */
+    override get level(): number {
+        return this.campaign?.level ?? super.level;
```

Comment:
ok but isn't the level a kingdom level?

Reply:
If i recall, the reason is that parties dont have a level in any sense and it came in handy for certain things (such as for the feat group which takes an actor).

Is there something you had in mind regarding parties having a level. I can get around it if I allow actor property overrides to places. I might need to be clever for statistics "anyways".

Reply:
Yeah, campaign.kingdom was just an alternative if we wanted to support multiple campaigns for a party. May or may not be a good thing given modules will eventually be able to extend it (though its not on my high priority list, but I did design it so that its possible later).